hack/api-docs/requirements.txt

```diff
@@ -20,7 +20,7 @@ mkdocs==1.5.3
 mkdocs-macros-plugin==1.0.5
 mkdocs-material==9.5.7
 mkdocs-material-extensions==1.3.1
-mkdocs-minify-plugin==0.7.2
+mkdocs-minify-plugin==0.8.0
```

Comment:
They didn't create a release for the version but [the change itself](https://github.com/byrnereese/mkdocs-minify-plugin/pull/37) seems fine to me in the sense that it does not contain any breaking changes.